src/Core/Services/Implementations/EventService.cs

```diff
@@ -222,6 +223,10 @@ public async Task LogOrganizationEventAsync(Organization organization, EventType
             await _eventWriteService.CreateAsync(e);
         }
 
+        public Task LogProviderUserEventAsync(ProviderUser providerUser, EventType type, DateTime? date = null) => throw new NotImplementedException();
```

Comment:
Should this just return `Task.Completed` or is throwing an exception appropriate here?